src/app/tree-grid/tree-grid-summary2-sample/tree-grid-summary2-sample.component.html

```diff
@@ -12,7 +12,7 @@
 
 <igx-tree-grid [igxPreventDocumentScroll]="true"  #treegrid1 [data]="data" [autoGenerate]="false" height="500px" width="100%"
     primaryKey="ID" foreignKey="ParentID">
-    <igx-column #col field="ID" header="Product ID" [sortable]="true">
+    <igx-column #col field="ID" header="Order ID" [sortable]="true">
     </igx-column>
     <igx-column #col field="Name" header="Product Name" [hasSummary]="true">
```

Comment:
Fix this and the `Discontinued` and `Added Date`.

```suggestion
    <igx-column #col field="Name" header="Order Name" [hasSummary]="true">
```